Misc/Linux/com.openutau.OpenUtau.metainfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.openutau.OpenUtau</id>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+
+  <name>OpenUtau</name>
+  <summary>Open singing synthesis platform</summary>
+
+  <developer id="io.github.stakira">
+    <name>StAkira</name>
+  </developer>
+
+  <description>
+    <p>OpenUtau is a free, open-source editor made for the UTAU community, featuring a modern user experience and support for multiple vocal tracks.</p>
+  </description>
+
+  <launchable type="desktop-id">com.openutau.OpenUtau.desktop</launchable>
+
+  <content_rating type="oars-1.1" />
+
+  <url type="bugtracker">https://github.com/stakira/OpenUtau/issues</url>
+  <url type="homepage">https://www.openutau.com/</url>
+  <url type="vcs-browser">https://github.com/stakira/OpenUtau</url>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/user-attachments/assets/bbb5bcba-ce56-4312-b122-1f49351cd8dc</image>
```

Comment:
I suggest this screenshot of the piano roll to better represent OpenUtau

https://framerusercontent.com/images/r0vjnljLyJ3LYJexkBazkbjDhNg.png